testserv.c

```diff
@@ -55,7 +55,7 @@ muststart(char *a0, char *a1, char *a2, char *a3, char *a4)
 
     /* now in child */
 
-    execlp(a0, a0, a1, a2, a3, a4, NULL);
+    execlp(a0, a0, a1, a2, a3, a4, (char*)NULL);
```

Comment:
The error thrown here without this change is:
![beanstalkd beanstalkd-build 1 2016-04-03 21-31-38](https://cloud.githubusercontent.com/assets/1804/14237602/7be19ed0-f9e3-11e5-8e6a-42b01f112c0e.png)
This is with GCC 4.7.4 on SmartOS joyent_20160317T000621Z.